WooFoundation/WooFoundation/ViewModifiers/WooStyleModifiers.swift

```diff
@@ -128,10 +128,12 @@ public struct ErrorStyle: ViewModifier {
     }
 }
 
+// The color of the bar button items in the navigation bar
+//
 public struct WooNavigationBarStyle: ViewModifier {
     public func body(content: Content) -> some View {
         content
-            .accentColor(Color(.accent)) // The color of bar button items in the navigation bar
+            .tint(Color(.accent))
```

Comment:
Since accentColor is not technically fully deprecated until iOS16, here we could also do something like:

```
        if #available(iOS 15.0, *) {
            return content.tint(Color(.accent))
        } else {
            return content.accentColor(Color(.accent))
        }
````
But since our minimum supported version is iOS15 anyway, I just went ahead with the more straight forward change so we don't have to remove it later. Happy to change this if you feel it's necessary.